spec/system/partner_system_spec.rb

```diff
@@ -50,6 +50,33 @@
     end
   end
 
+  describe 'adding another user to a partner account' do
```

Comment:
Does this test do the same thing as in line 80 here? If so, can we remove the other one in favor of this one?

Reply:
I'll have to take a look. Honestly not too sure. They appear to be written up step-wise as two slightly different workflows. They might be fundamentally doing the same stuff though. I'll have to check it out in a browser to be sure. If they are fundamentally the same then yeah it's certainly worth flattening into a single test.

Reply:
It looks like the test starting on line 53 adds a new user to an existing partner and the test starting on line 80 adds a new partner and their first user.